### ISSUE TITLE
Calculating back to actual weights of loss functions

### ISSUE THREAD

**Issue by edshkim98** [yaringal/multi-task-learning-example#15]
@yaringal  Hi, thank you very much for releasing the code here!
I am currently using your technique in adversarial loss training for semantic segmentation.
However, I would like to know the actual weights applying to each loss function (Cross Entropy, Adversarial Loss).
Could you please help me with how to get these weights?

(I am currently using your equation of std and do 1/(2*std**2) to get the weight. Is this correct?)
